apm-sniffer/config/agent.config

```diff
@@ -38,6 +38,9 @@ agent.service_name=${SW_AGENT_NAME:Your_ApplicationName}
 # Skywalking team may ask for these files in order to resolve compatible problem.
 # agent.is_open_debugging_class = ${SW_AGENT_OPEN_DEBUG:true}
 
+# The operationName max length
+# agent.operation_name_threshold=${OPERATION_NAME_THRESHOLD:500}
```

Comment:
The agent set up document requires to update. Add this, and notice user, > 500 is not recommended.

Reply:
ok, I have finished.

Reply:
Is it better to prefixed it with something like `SW_AGENT_`?, we have this prefix in other configs

Reply:
Make sense to me, please adjust to the ENV name prefix.

Reply:
Sorry, I'm going to change it now

Reply:
done